public/resources/js/check_session.php

```diff
@@ -0,0 +1,11 @@
+<script>
+function checkSession() {
+    <?php if(!isset($_SESSION['user']['id'])): ?>
+    while (true) {
+        if (confirm("You need to log in to perform this action.")) {
+            break; 
+        }
+    }
+    <?php endif; ?>
+}
+</script>
```

Comment:
this file is useless, becuase you already have the session in the app objs, so you can check this in any css file, plus the way you make a warning is wrong, we are using sweetalert2 to make the alerts, so make an alert with this API and delete this file since it is not a js script